src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+      {person.age ? (
+        <p className="Person__age">{`I am ${person.age}`}</p>
+      ) : null}
```

Comment:
1. Better use the following:
```suggestion
      {person.age  && <p className="Person__age">{`I am ${person.age}`}</p>}
```

2. Both (mine and yours) code samples have the problem for the person, with `age===0`. Please fix it yourself.

Reply:
Thx for ur response . i will fix that ! =)